pkg/cmd/run.go

```diff
@@ -782,15 +788,15 @@ func resolvePodTemplate(ctx context.Context, cmd *cobra.Command, templateSrc str
 	return err
 }
 
-func uploadFileOrDirectory(platform *v1.IntegrationPlatform, item string, integrationName string, cmd *cobra.Command, integration *v1.Integration) error {
+func uploadFileOrDirectory(platform *v1.IntegrationPlatform, item string, integrationName string, cmd *cobra.Command, integration *v1.Integration, verbose bool) error {
```

Comment:
Maybe we can change it into method of `runCmdOptions` as the args now looks a bit too long.